pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.ruff]
+lint.select = ["PL", "YTT", "BLE", "I", "UP", "A", "COM", "T10", "EXE", "TID", "F401", "F402", "F821"]
+
+lint.ignore = ["E501", "PLR2004", "COM812", ]
+
+target-version = "py310"
+
+extend-exclude = [
+    "docker/alternator-dns",
```

Comment:
Why is it exclusion?
I think we should ad to exclusion only those files which cannot/shouldnot change.
Such as imported `as-is` from some projects or designed to satisfy some tools/libraries which don't know about ruff preferences...

Reply:
the file originates from: https://github.com/scylladb/alternator-load-balancing/blob/master/dns/dns-loadbalancer.py

so we didn't wanted to do many changes for, so it was excluded from linting and auto formatting.
so it can be comparabled, but never had any changes, and not sure what's the plans for it

Reply:
The comment about the source of the file actually should have been in the file itself or text file around it.
Example: https://github.com/scylladb/scylla-cluster-tests/blob/d92d7cc71665c6155bea717ca7082723861d7183/sdcm/utils/version_utils.py#L48

Reply:
I've removed it from `exclude`

we don't really update this file ever, we can format it as we want.